README.rst

```diff
@@ -53,14 +51,14 @@ Development
 .. _`P1 Bugs`: https://mzl.la/2BGZy44
 .. _`P2 Bugs`: https://mzl.la/2oniZa3
 .. _`All developer.mozilla.org bugs`: https://mzl.la/2onLvZ8
-.. _`Pull Request Queues`: http://prs.mozilla.io/mozilla:kuma%3Bmdn:kumascript,infra,mdn-fiori
-.. _`What's Deployed on MDN?`: https://whatsdeployed.io/s/HC0/mozilla/kuma
+.. _`Pull Request Queues`: http://prs.mozilla.io/mdn:kuma,kumascript,infra,mdn-fiori
```

Comment:
'%3Bmdn:' was replaced by a comma on line56. is that right?

Reply:
The line that remains works.